addons/web/static/tests/setup.js

```diff
@@ -13,6 +14,39 @@ const { whenReady, loadFile } = owl.utils;
 owl.config.enableTransitions = false;
 owl.QWeb.dev = true;
 
+function stringifyObjectValues(obj, properties) {
+    let res = "";
+    for (const dotted of properties) {
+        const keys = dotted.split(".");
+        let val = obj;
+        for (const k of keys) {
+            val = val[k];
```

Comment:
oups... there is an error here... this probably does not do what you expect (when there are more than 1 properties...)

Reply:
nevermind